config/locales/en/errors.yml

```diff
@@ -323,6 +323,17 @@ en:
               inclusion: Select how your client manages with no income
             manage_other_details:
               blank: Enter details on how your client manages with no income
+        steps/outgoings/housing_payment_type_form:
+          attributes:
+            housing_payment_type:
+              inclusion: Select which payment your client pays where they usually live
+        steps/outgoings/council_tax_form:
+          attributes:
+            pays_council_tax:
+              inclusion: Select yes if your client pays Council Tax where they usually live
+            council_tax_amount:
+              blank: Enter how much your client pays yearly
+              not_a_number: How much they pay yearly must be a number, like 50
```

Comment:
I don't know how to actually hit this as a user, especially with the field in the html set as a `govuk_number_field`, _and_ `inputmode: 'numeric'` 
 It does show if I remove `presence: true ` from the `council_tax_amount` validation in the form
 
I also can't hit it in the spec when passing in a string. 

Don't think this is a huge issue as the form UI doesn't allow for non-numeric input

Reply:
I don't think you can get to it via the UI